### PR TITLE
J’agis FNV / Sur la page de fin [NGC-1258] 

### PR DESCRIPTION
**Pull request by florianpanchout**

### CODE REVIEW

File: src/hooks/simulation/useSendSimulationToAgir.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import { SERVER_URL } from '@/constants/urls'
+import { useCurrentSimulation } from '@/publicodes-state'
+import { useMutation } from '@tanstack/react-query'
+import axios from 'axios'
+
+export function useSendSimulationToAgir() {
```
Comment (paulsouche) [inline src/hooks/simulation/useSendSimulationToAgir.ts:6]: As the enpoint is `/export-situation` what do you think of `useExportSituationToAgir` ?

File: src/app/(simulation)/(large-layout-nosticky)/fin/_components/AgirSecondaryBlock.tsx [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+
+export default function AgirSecondaryBlock() {
+  const { t } = useClientTranslation()
+
+  const { sendSimulation, data, isPending, isSuccess, isError, error } =
+    useSendSimulationToAgir()
+
+  useEffect(() => {
+    if (data?.redirectUrl && isSuccess) {
+      window.open(data.redirectUrl, '_blank')
```
Comment (paulsouche) [inline src/app/(simulation)/(large-layout-nosticky)/fin/_components/AgirSecondaryBlock.tsx:20]: I don't think you can do this. It should be in the `click` life cycle 

https://stackoverflow.com/questions/14643040/allow-window-open-to-open-new-window-and-not-popup

Comment (florianpanchout) [reply src/app/(simulation)/(large-layout-nosticky)/fin/_components/AgirSecondaryBlock.tsx:20]: The browser decision is [apparently based on the delay between a click and the opening](https://www.mikepalmer.dev/blog/open-a-new-window-without-triggering-pop-up-blockers). Here, you click on the button before opening the page. I tested on Chrome (desktop and mobile), Safari and Firefox without issues

Comment (paulsouche) [reply src/app/(simulation)/(large-layout-nosticky)/fin/_components/AgirSecondaryBlock.tsx:20]: Ok fair enough !

**Review by paulsouche — APPROVED** [review]

**Comment by bjlaa** [comment]
il te manque des trads !

**Review by bjlaa — APPROVED** [review]

**Pull request merged**